### PR TITLE
Add NYC HRA logo to header

### PR DESCRIPTION
**Pull request by tdooner**
## Ticket

Resolves FFS-1350.

## Changes

This also makes the page's min-width a bit larger to compensate for the
width of this logo.

## Context for reviewers

N/A

## Testing

Acceptance test screenshots.

### CODE REVIEW

**Comment by tdooner** [comment]
@carmenrosalop Screenshots below:

Screenshot 2024-08-28 at 3 26 07 PM

Screenshot 2024-08-28 at 3 26 13 PM

**Review by carmenrosalop — COMMENTED** [review]
Thanks Tom! That was quick. I'm wondering if for the mobile version we can make the NYC logo a bit smaller so that it doesn't overpower SNAP Income Pilot?

File: app/app/assets/stylesheets/cbv.scss [hunk 1]
```diff
@@ -63,6 +63,16 @@ html {
...
   height: 45px;
 }
+.cbv-header__agency-logo--ma {
+  height: 45px;
+}
+.cbv-header__agency-logo--nyc {
+  height: 35px;
+
+  @include at-media($theme-header-min-width) {
+    height: 25px;
```
[the remark this answers is not in the document]
Comment (tdooner) [reply app/app/assets/stylesheets/cbv.scss:73]: That's the desktop size, the mobile one is the 35px

**Comment by tdooner** [comment]
> Also, are you implementing only the client-facing header? or the staff one too?

@carmenrosalop It will be both. The headers are the same across the caseworker pages and the applicant-facing pages.

**Comment by carmenrosalop** [comment]
@tdooner I was asking about the staff facing header because I wanted to see how the log in / log out text looks w the logo

**Comment by tdooner** [comment]
@carmenrosalop 
image

image

**Review by carmenrosalop — APPROVED** [review]
Looks great!

**Pull request merged**